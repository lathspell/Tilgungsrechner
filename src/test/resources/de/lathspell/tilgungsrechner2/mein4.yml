# Mit KfW Rechner verifiziert
# Die Cents stimmen nicht immer, vermutlich rundet der Webrechner anders
months:
    "2016-05":
        darlehensbetrag: 5000
        sollzins: 0.75
        monatsrate: -1
        auszahlung: 5000
    "2018-05":
        # 10% vom Darlehensbetrag
        tilgungszuschuss: 500
        monatsrate: 53.68
    "2026-05":
        zinsbindung: true


    