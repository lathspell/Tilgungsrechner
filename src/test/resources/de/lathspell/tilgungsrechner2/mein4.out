# mein4
Tilgungsplan
Periode 	Rate (EUR) 	Zinsanteil (EUR) 	Tilgungsanteil (EUR) 	Restschuld am Periodenende (EUR)
Mai 2016             3,13               3,13                   0,00                  5.000,00
Jun 2016             3,13               3,13                   0,00                  5.000,00
Jul 2016             3,13               3,13                   0,00                  5.000,00
Aug 2016             3,13               3,13                   0,00                  5.000,00
Sep 2016             3,13               3,13                   0,00                  5.000,00
Okt 2016             3,13               3,13                   0,00                  5.000,00
Nov 2016             3,13               3,13                   0,00                  5.000,00
Dez 2016             3,13               3,13                   0,00                  5.000,00
Jan 2017             3,13               3,13                   0,00                  5.000,00
Feb 2017             3,13               3,13                   0,00                  5.000,00
Mär 2017             3,13               3,13                   0,00                  5.000,00
Apr 2017             3,13               3,13                   0,00                  5.000,00
Mai 2017             3,13               3,13                   0,00                  5.000,00
Jun 2017             3,13               3,13                   0,00                  5.000,00
Jul 2017             3,13               3,13                   0,00                  5.000,00
Aug 2017             3,13               3,13                   0,00                  5.000,00
Sep 2017             3,13               3,13                   0,00                  5.000,00
Okt 2017             3,13               3,13                   0,00                  5.000,00
Nov 2017             3,13               3,13                   0,00                  5.000,00
Dez 2017             3,13               3,13                   0,00                  5.000,00
Jan 2018             3,13               3,13                   0,00                  5.000,00
Feb 2018             3,13               3,13                   0,00                  5.000,00
Mär 2018             3,13               3,13                   0,00                  5.000,00
Apr 2018             3,13               3,13                   0,00                  5.000,00
>> Neuer Tilgungssatz 12,13 % ab Mai 2018 (neue Rate 53,68)
Mai 2018            53,68               2,81                  50,87                  4.449,13
>> Tilgungszuschuss: 500,00
Jun 2018            53,68               2,78                  50,90                  4.398,23
Jul 2018            53,68               2,75                  50,93                  4.347,30
Aug 2018            53,68               2,72                  50,96                  4.296,34
Sep 2018            53,68               2,69                  50,99                  4.245,35
Okt 2018            53,68               2,65                  51,03                  4.194,32
Nov 2018            53,68               2,62                  51,06                  4.143,26
Dez 2018            53,68               2,59                  51,09                  4.092,17
Jan 2019            53,68               2,56                  51,12                  4.041,05
Feb 2019            53,68               2,53                  51,15                  3.989,90
Mär 2019            53,68               2,49                  51,19                  3.938,71
Apr 2019            53,68               2,46                  51,22                  3.887,49
Mai 2019            53,68               2,43                  51,25                  3.836,24
Jun 2019            53,68               2,40                  51,28                  3.784,96
Jul 2019            53,68               2,37                  51,31                  3.733,65
Aug 2019            53,68               2,33                  51,35                  3.682,30
Sep 2019            53,68               2,30                  51,38                  3.630,92
Okt 2019            53,68               2,27                  51,41                  3.579,51
Nov 2019            53,68               2,24                  51,44                  3.528,07
Dez 2019            53,68               2,21                  51,47                  3.476,60
Jan 2020            53,68               2,17                  51,51                  3.425,09
Feb 2020            53,68               2,14                  51,54                  3.373,55
Mär 2020            53,68               2,11                  51,57                  3.321,98
Apr 2020            53,68               2,08                  51,60                  3.270,38
Mai 2020            53,68               2,04                  51,64                  3.218,74
Jun 2020            53,68               2,01                  51,67                  3.167,07
Jul 2020            53,68               1,98                  51,70                  3.115,37
Aug 2020            53,68               1,95                  51,73                  3.063,64
Sep 2020            53,68               1,91                  51,77                  3.011,87
Okt 2020            53,68               1,88                  51,80                  2.960,07
Nov 2020            53,68               1,85                  51,83                  2.908,24
Dez 2020            53,68               1,82                  51,86                  2.856,38
Jan 2021            53,68               1,79                  51,89                  2.804,49
Feb 2021            53,68               1,75                  51,93                  2.752,56
Mär 2021            53,68               1,72                  51,96                  2.700,60
Apr 2021            53,68               1,69                  51,99                  2.648,61
Mai 2021            53,68               1,66                  52,02                  2.596,59
Jun 2021            53,68               1,62                  52,06                  2.544,53
Jul 2021            53,68               1,59                  52,09                  2.492,44
Aug 2021            53,68               1,56                  52,12                  2.440,32
Sep 2021            53,68               1,53                  52,15                  2.388,17
Okt 2021            53,68               1,49                  52,19                  2.335,98
Nov 2021            53,68               1,46                  52,22                  2.283,76
Dez 2021            53,68               1,43                  52,25                  2.231,51
Jan 2022            53,68               1,39                  52,29                  2.179,22
Feb 2022            53,68               1,36                  52,32                  2.126,90
Mär 2022            53,68               1,33                  52,35                  2.074,55
Apr 2022            53,68               1,30                  52,38                  2.022,17
Mai 2022            53,68               1,26                  52,42                  1.969,75
Jun 2022            53,68               1,23                  52,45                  1.917,30
Jul 2022            53,68               1,20                  52,48                  1.864,82
Aug 2022            53,68               1,17                  52,51                  1.812,31
Sep 2022            53,68               1,13                  52,55                  1.759,76
Okt 2022            53,68               1,10                  52,58                  1.707,18
Nov 2022            53,68               1,07                  52,61                  1.654,57
Dez 2022            53,68               1,03                  52,65                  1.601,92
Jan 2023            53,68               1,00                  52,68                  1.549,24
Feb 2023            53,68               0,97                  52,71                  1.496,53
Mär 2023            53,68               0,94                  52,74                  1.443,79
Apr 2023            53,68               0,90                  52,78                  1.391,01
Mai 2023            53,68               0,87                  52,81                  1.338,20
Jun 2023            53,68               0,84                  52,84                  1.285,36
Jul 2023            53,68               0,80                  52,88                  1.232,48
Aug 2023            53,68               0,77                  52,91                  1.179,57
Sep 2023            53,68               0,74                  52,94                  1.126,63
Okt 2023            53,68               0,70                  52,98                  1.073,65
Nov 2023            53,68               0,67                  53,01                  1.020,64
Dez 2023            53,68               0,64                  53,04                    967,60
Jan 2024            53,68               0,60                  53,08                    914,52
Feb 2024            53,68               0,57                  53,11                    861,41
Mär 2024            53,68               0,54                  53,14                    808,27
Apr 2024            53,68               0,51                  53,17                    755,10
Mai 2024            53,68               0,47                  53,21                    701,89
Jun 2024            53,68               0,44                  53,24                    648,65
Jul 2024            53,68               0,41                  53,27                    595,38
Aug 2024            53,68               0,37                  53,31                    542,07
Sep 2024            53,68               0,34                  53,34                    488,73
Okt 2024            53,68               0,31                  53,37                    435,36
Nov 2024            53,68               0,27                  53,41                    381,95
Dez 2024            53,68               0,24                  53,44                    328,51
Jan 2025            53,68               0,21                  53,47                    275,04
Feb 2025            53,68               0,17                  53,51                    221,53
Mär 2025            53,68               0,14                  53,54                    167,99
Apr 2025            53,68               0,10                  53,58                    114,41
Mai 2025            53,68               0,07                  53,61                     60,80
Jun 2025            53,68               0,04                  53,64                      7,16
Jul 2025             7,16               0,00                   7,16                      0,00
GESAMT: 111 Monatsraten: 4.623,64   davon Zinsen: 198,76  für ein Darlehen über 5.000,00
